pure_changelog.md

```diff
@@ -12,6 +12,7 @@
 * Fixed MTP issues
 
 ### Fixed
+* Fixed SIM cards window
```

Comment:
I'd rather say it's way more than a fix - that's kind of a rebuild because the user flows have changed.

Reply:
done